attributes/conditions.rb

```diff
@@ -22,3 +22,8 @@
   || node['platform'] == 'amazon' || node['platform'] == 'ubuntu'
 
 default['conditions']['ami_bootstrapped'] = ami_bootstrapped?
+
+# alinux1 and centos6 use an old cgroup directory: /cgroup, all other OSs use /sys/fs/cgroup, which is the default
+# "platform_verision" on alinux1 is "2018.03" and "platform_verision" on alinux2 is "2"
+default['conditions']['old_cgroup_directory'] = (node['platform'] == 'centos' && node['platform_version'].to_i == 6) \
+  || (node['platform'] == 'amazon' && node['platform_version'] != "2")
```

Comment:
can we match alinux1 here `node['platform_version'] != "2"` so that it is forward compatible?